ColabTool/Shared/PostTopDetails/story.jsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import {
+  storiesOf,
+} from '@kadira/storybook';
+import PostTopDetails from './index';
+
+const draftDetails = {
+  userName: 'Ash',
+  avatarUrl: 'https://buffer-uploads.s3.amazonaws.com/510521020a19000b6a00001e/a476fed03b1de4e06563d6063d7d3ee0.jpg',
+  email: 'ash@buffer.com',
+  via: 'web',
+  createdAt: 'March 2nd at 12:45pm (GMT)',
+  postAction: 'This post is scheduled for 9:42pm (GMT)',
+};
+
+const justCreatedTimeDraftDetails = {
+  userName: 'Ash',
+  avatarUrl: 'https://buffer-uploads.s3.amazonaws.com/510521020a19000b6a00001e/a476fed03b1de4e06563d6063d7d3ee0.jpg',
+  email: 'ash@buffer.com',
+  via: 'web',
+  createdAt: 'at 12:45pm (GMT)',
+  postAction: 'This post is scheduled for 9:42pm (GMT)',
+};
+
+const noUserNameDraftDetails = {
+  userName: '',
+  avatarUrl: 'https://buffer-uploads.s3.amazonaws.com/510521020a19000b6a00001e/a476fed03b1de4e06563d6063d7d3ee0.jpg',
+  email: 'ash@buffer.com',
+  via: 'web',
+  createdAt: 'March 2nd at 12:45pm (GMT)',
+  postAction: 'This post is scheduled for 9:42pm (GMT)',
+};
+
+const noUserNameJustTimeDraftDetails = {
+  userName: '',
+  avatarUrl: 'https://buffer-uploads.s3.amazonaws.com/510521020a19000b6a00001e/a476fed03b1de4e06563d6063d7d3ee0.jpg',
+  email: 'ash@buffer.com',
+  via: 'web',
+  createdAt: 'at 12:45pm (GMT)',
+  postAction: 'This post is scheduled for 9:42pm (GMT)',
+};
+
+const noNameOrEmailDraftDetails = {
+  userName: '',
```

Comment:
Heya @hharnisc - good one 😄  I'm not sure but after chatting with @tomredman - we thought it would be good to have a fall-back in case there isn't one.